WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewView.swift

```diff
@@ -193,12 +264,19 @@ private extension ThemesPreviewView {
             value: "Unable to render the demo for this theme. Please try another theme.",
             comment: "The error message shown if the app can't show a theme demo."
         )
+
+        static let pagesSheetHeading = NSLocalizedString(
+        "ThemesPreviewView.pagesSheetHeading",
+        value: "View other store pages on this theme",
+        comment: "Heading for sheet displaying list of pages"
+        )
```

Comment:
nit: Code indentation needs to be updated.

Reply:
right! Added in https://github.com/woocommerce/woocommerce-ios/pull/11492/commits/16671b37bad6fb0a0c6a544add412a6d0739eb26